docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -0,0 +1,1071 @@
+---
+id: dual-region-operational-procedure
+title: "Dual-region operational procedure"
+sidebar_label: "Dual-region operational procedure"
+description: "The operational procedure concerning dual-region setups to recover from a region loss."
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+import StateContainer from './components/stateContainer.jsx';
+
+<!-- Failover -->
+
+import Three from './img/3.svg';
+import Four from './img/4.svg';
+import Five from './img/5.svg';
+import Six from './img/6.svg';
+import Seven from './img/7.svg';
+
+<!-- Failback -->
+
+import Nine from './img/9.svg';
+import Ten from './img/10.svg';
+import Eleven from './img/11.svg';
+import Twelve from './img/12.svg';
+import Thirteen from './img/13.svg';
+import Fourteen from './img/14.svg';
+import Fifteen from './img/15.svg';
+
+## Introduction
+
+This operational procedure is a step-by-step guide on how to restore operations in the case of a total region failure. It explains how to temporarily restore functionality in the surviving region, and how to ultimately do a full recovery to restore the dual-region setup. The operational procedure builds on top of the [dual-region AWS setup guide](./../../platform-deployment/helm-kubernetes/platforms/amazon-eks/dual-region.md), but is generally applicable for any dual-region setup.
+
+Before proceeding with the operational procedure, you must thoroughly read and comprehend the contents of the [dual-region concept page](./../../concepts/multi-region/dual-region.md). This page outlines various limitations and requirements pertinent to the procedure, which are crucial for successful execution.
+
+## Disclaimer
+
+:::danger
+
+- Customers must develop and test the below-described operational procedure in non-production environments based on the framework steps outlined by Camunda, **before applying them in production setups**.
+- Before advancing to production go-live, validating these procedures with Camunda is strongly recommended.
+- Customers are solely responsible for detecting any regional failures and implementing the necessary described operational procedure.
+
+:::
+
```

Comment:
Let's add here a terminology section (or similar) where we explain what are:
- Surviving region
- Lost region
- Recreated region 

Even though the first two are intuitive, I think it's important to set the terms in the beginning explicitly, since they'd be used across the procedure doc later.

Reply:
or in the concept doc http://localhost:3000/docs/next/self-managed/concepts/multi-region/dual-region/#region-loss if it makes more sense to declare them there.

Reply:
I'll add it as part of the operational procedure its own "chapter".
I think adding to the concept page could cause more confusion since you already have active passive and then throwing more in the mix for something similar is probably not going well.